app/assets/stylesheets/application.css.scss

```diff
@@ -160,3 +160,19 @@ i.fa.medium {
 ::-webkit-input-placeholder {
   color: $placeholder-text;
 }
+
+// makes vimeo videos responsive
+.video-responsive {
```

Comment:
@sethbergman Looks like some of the other videos (`home.html.erb` for example), are using the class `.video-container` from Materialize CSS to enable responsive embeds: http://materializecss.com/media-css.html#videos.

Haven't done any ad-hoc testing myself, so I dunno if the above `.video-responsive` class improves on the Materialize one?